engine/runner/src/main/java/org/enso/runner/EnsoLibraryFeature.java

```diff
@@ -0,0 +1,93 @@
+package org.enso.runner;
+
+import static scala.jdk.javaapi.CollectionConverters.asJava;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.LinkedHashSet;
+import java.util.TreeSet;
+import org.enso.compiler.core.ir.module.scope.imports.Polyglot;
+import org.enso.pkg.PackageManager$;
+import org.graalvm.nativeimage.hosted.Feature;
+import org.graalvm.nativeimage.hosted.RuntimeReflection;
+
+public final class EnsoLibraryFeature implements Feature {
+  @Override
+  public void beforeAnalysis(BeforeAnalysisAccess access) {
+    var libs = new LinkedHashSet<Path>();
+    for (var p : access.getApplicationClassPath()) {
```

Comment:
@hubertp, @Akirathan is there a better way to get all libraries in `Standard.*` namespace? We are in `runner` - e.g. we should be able to see everything the `org.enso.runner.Main` sees.

If there was a way to iterate all libraries in the distribution, we could provide a property to list a comma separated list of libraries to include in the native executable and then just parse those listed.

Reply:
I can't think of any, without the dependency on `runtime`. With dependency on `runtime`, we could create `org.graalvm.polyglot.Context` from which we can get `EnsoContext` and have access to all the libraries.

Reply:
OK, let's leave it as it is for now. The `runtime-compiler` needs to know all libraries and their sources, it'd be good if that was unchained from `runtime` itself.